nautobot/docs/user-guides/custom-fields.md

```diff
@@ -0,0 +1,344 @@
+# Custom Fields
+
+Custom fields are a method of adding new fields to existing Nautobot models. For more general information on how custom fields work, refer to the [custom fields model documentation](../models/extras/customfield.md).
```

Comment:
If we are going to go with two separate custom-field documentation pages, then `models/extras/customfield.md` should also have a link back to this new file to aid with discovery.